### ISSUE TITLE
Uncaught Error: Class 'External_User_Token' not found

### ISSUE THREAD

**Issue by edent** [indieweb/wordpress-indieauth#224]
Seeing lots of these errors appear since the update.

```
[05-Oct-2021 09:52:01 UTC] PHP Fatal error:  Uncaught Error: Class 'External_User_Token' not found in /shkspr.mobi/blog/wp-content/plugins/indieauth/indieauth.php:85
Stack trace:
#0 /shkspr.mobi/blog/wp-includes/class-wp-hook.php(303): IndieAuth_Plugin::expires(false)
#1 /shkspr.mobi/blog/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#2 /shkspr.mobi/blog/wp-includes/plugin.php(518): WP_Hook->do_action(Array)
#3 /shkspr.mobi/blog/wp-cron.php(138): do_action_ref_array('indieauth_clean...', Array)
#4 {main}
  thrown in /shkspr.mobi/blog/wp-content/plugins/indieauth/indieauth.php on line 85
```

**Comment by dshanske** [comment]
That's odd, file is there.. https://github.com/indieweb/wordpress-indieauth/blob/trunk/includes/class-external-token.php

**Comment by dshanske** [comment]
I'll fix it tonight

**Comment by dshanske** [comment]
Resolved in 4.1.1
